libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/ExecSpecRule/Conway/Certs.hs

```diff
@@ -0,0 +1,39 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE TypeOperators #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Test.Cardano.Ledger.Conformance.ExecSpecRule.Conway.Certs () where
+
+import Cardano.Ledger.Conway
+import Data.Bifunctor (first)
+import qualified Data.List.NonEmpty as NE
+import qualified Data.Text as T
+import qualified Lib as Agda
+import Test.Cardano.Ledger.Conformance
+import Test.Cardano.Ledger.Conformance.ExecSpecRule.Conway.Base
+import Test.Cardano.Ledger.Constrained.Conway
+
+instance
+  IsConwayUniv fn =>
+  ExecSpecRule fn "CERTS" Conway
+  where
+  type ExecContext fn "CERTS" Conway = ConwayCertExecContext Conway
+
+  environmentSpec _ = certsEnvSpec
+
+  stateSpec _ _ = certStateSpec
+
+  signalSpec _ _ _ = txCertsSpec
+
```

Comment:
Out of curiosity, where are these things defined?
txCertsSpec is going to need some special propoerties
1) No duplicates allowed
2) Later Certs in the sequence must be aware of Certs earlier in the sequence.

Reply:
They are defined here: https://github.com/IntersectMBO/cardano-ledger/blob/9c94eddede4400e2d3ec926cfeec3a37bfa33b1f/libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Conway/Certs.hs
Also, thanks for pointing out the above.